entities/Order.go

```diff
@@ -58,3 +58,17 @@ func (o *Order) InitStatus() (*Order, error) {
 
 	return o, fmt.Errorf("%w: from %s to %s", errors.New("invalid order status"), o.Status, "New")
 }
+
+func (o *Order) IsEqualCreatedOrder(createdOrder *Order) bool {
```

Comment:
Why you have or use Func IsEqual+service+entities ?

Reply:
now this function was deleted

Reply:
Fixed it. Please check again.